### PR TITLE
Sample submissions for Cmdlets -AJSING

### PR DESCRIPTION
**Pull request by ajsing**
## Description
Sample for below Application Autoscaling PowerShell cmdlets:
Get-AASScalableTarget
Get-AASScalingActivity
Get-AASScalingPolicy
Remove-AASScalableTarget
Add-AASScalableTarget
Set-AASScheduledAction
Get-AASScheduledAction
Remove-AASScalingPolicy
Remove-AASScheduledAction

## Motivation and Context
Sample is Missing.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist
- [ ] My code follows the code style of this project
- [x ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [ ] I have read the **README** document
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed

## License
- [ x] I confirm that this pull request can be released under the Apache 2 license

[issues]: https://github.com/aws/aws-tools-for-powershell/issues
[license]: http://aws.amazon.com/apache2.0/
[cla]: http://en.wikipedia.org/wiki/Contributor_License_Agreement

### CODE REVIEW

File: generator/AWSPSGeneratorLib/HelpMaterials/Examples/EKS/Add-EKSResourceTag.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--Examples for the AWS PowerShell CmdLets-->
+<examples>
+  <example>
+    <code>PS C:\> $EKSTags = @{Name = "EKSPRODCLUSTER"}
+  PS C:\> Add-EKSResourceTag -ResourceArn "arn:aws:eks:us-west-2:012345678912:cluster/PROD" -Tag System.Collections.Hashtable</code>
```
Comment (ketanbhut) [inline generator/AWSPSGeneratorLib/HelpMaterials/Examples/EKS/Add-EKSResourceTag.xml]: > -Tag System.Collections.Hashtable</code>
Looks like formatting ..- should be $EKSTags

File: generator/AWSPSGeneratorLib/HelpMaterials/Examples/EKS/Update-EKSClusterConfig.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--Examples for the AWS PowerShell CmdLets-->
+<examples>
+  <example>
+    <code>PS C:\&gt; Amazon.EKS.Model.LogSetup =  New-Object Amazon.EKS.Model.LogSetup -Property @{Types="api","audit","authenticator","controllerManager","scheduler",Enabled="True"}
```
Comment (ketanbhut) [inline generator/AWSPSGeneratorLib/HelpMaterials/Examples/EKS/Update-EKSClusterConfig.xml]: `PS C:\&gt; Amazon.EKS.Model.LogSetup =`
> is replaced with &gt

File: generator/AWSPSGeneratorLib/HelpMaterials/Examples/EKS/Update-EKSClusterConfig.xml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+PS C:\&gt; Update-EKSClusterConfig -Name "PROD" -Logging_ClusterLogging Amazon.EKS.Model.LogSetup
```
Comment (ketanbhut) [inline generator/AWSPSGeneratorLib/HelpMaterials/Examples/EKS/Update-EKSClusterConfig.xml]: `PS C:\&gt; Update-EKSClusterConfig`
Same as above

**Comment by ajsing** [comment]
Thanks Matteo for your valuable feedback, I optimized all the cmdlets as per your suggestions and updated PR. Please review and let me know if any further modification needed.

File: generator/AWSPSGeneratorLib/HelpMaterials/Examples/EKS/Get-EKSClusterList.xml [hunk 4]
```diff
@@ -2,11 +2,9 @@
 <!--Examples for the AWS PowerShell CmdLets-->
 <examples>
   <example>
-    <code>Get-EKSClusterList -Select *
-                                                                                                                                                                                            
-Clusters            NextToken
---------            ---------
-{DEV-KUBE-CL, PROD}</code>
+    <code>Get-EKSClusterList
+                                                                                                                                    PROD
```
Comment (matteo-prosperi) [inline generator/AWSPSGeneratorLib/HelpMaterials/Examples/EKS/Get-EKSClusterList.xml]: Why so many spaces before `PROD`? Is this actually the output of the cmdlet?

File: generator/AWSPSGeneratorLib/HelpMaterials/Examples/EKS/Update-EKSClusterConfig.xml [hunk 5]
```diff
@@ -2,9 +2,9 @@
 <!--Examples for the AWS PowerShell CmdLets-->
 <examples>
   <example>
-    <code>PS C:\> $LogSetup =  New-Object Amazon.EKS.Model.LogSetup -Property @{Types="api","audit","authenticator","controllerManager","scheduler",Enabled="True"}
+    <code>
 
-PS C:\> Update-EKSClusterConfig -Name "PROD" -Logging_ClusterLogging $LogSetup 
+PS C:\> Update-EKSClusterConfig -Name "PROD" -Logging_ClusterLogging @{Types="api","audit","authenticator","controllerManager","scheduler",Enabled="True"}
```
Comment (matteo-prosperi) [inline generator/AWSPSGeneratorLib/HelpMaterials/Examples/EKS/Update-EKSClusterConfig.xml]: Please remove the empty lines. This should look like
```
    <code>PS C:\> Update...
```

File: generator/AWSPSGeneratorLib/HelpMaterials/Examples/EKS/Add-EKSResourceTag.xml [hunk 6]
```diff
@@ -2,8 +2,8 @@
 <!--Examples for the AWS PowerShell CmdLets-->
 <examples>
   <example>
-    <code>PS C:\> $EKSTags = @{Name = "EKSPRODCLUSTER"}
-  PS C:\> Add-EKSResourceTag -ResourceArn "arn:aws:eks:us-west-2:012345678912:cluster/PROD" -Tag $EKSTags </code>
+    <code>
+  PS C:\> Add-EKSResourceTag -ResourceArn "arn:aws:eks:us-west-2:012345678912:cluster/PROD" -Tag @{Name = "EKSPRODCLUSTER"} </code>
```
Comment (matteo-prosperi) [inline generator/AWSPSGeneratorLib/HelpMaterials/Examples/EKS/Add-EKSResourceTag.xml]: Please remove the line break after `<code>`.
This should look like
```
    <code>PS C:\> Add...
```

File: generator/AWSPSGeneratorLib/HelpMaterials/Examples/ECS/Update-ECSClusterSetting.xml [hunk 7]
```diff
@@ -3,7 +3,7 @@
 <examples>
   <example>
     <code>
-Update-ECSClusterSetting -Cluster "LAB-ECS-CL" -Setting (New-Object Amazon.ECS.Model.ClusterSetting -Property @{Name="containerInsights"; Value="disabled"})                                                                                                                                                                        
+Update-ECSClusterSetting -Cluster "LAB-ECS-CL" -Setting @{Name="containerInsights"; Value="disabled"}
```
Comment (matteo-prosperi) [inline generator/AWSPSGeneratorLib/HelpMaterials/Examples/ECS/Update-ECSClusterSetting.xml]: Please remove the line break after `<code>`.
This should look like
```
    <code>Update...
```

File: generator/AWSPSGeneratorLib/HelpMaterials/Examples/ECS/Remove-ECSCluster.xml [hunk 8]
```diff
@@ -3,16 +3,14 @@
 <examples>
   <example>
     <code>
-Remove-ECSCluster -Cluster "LAB-ECS" -Select *
+Remove-ECSCluster -Cluster "LAB-ECS"
```
Comment (matteo-prosperi) [inline generator/AWSPSGeneratorLib/HelpMaterials/Examples/ECS/Remove-ECSCluster.xml]: Please remove the line break after `<code>`.
This should look like
```
    <code>Remove...
```

File: generator/AWSPSGeneratorLib/HelpMaterials/Examples/AAS/Set-AASScalingPolicy.xml [hunk 9]
```diff
@@ -3,22 +3,7 @@
...
-PS C:\> $stepadjustment.MetricIntervalUpperBound = 0
-PS C:\> $stepadjustment
-
-MetricIntervalLowerBound MetricIntervalUpperBound ScalingAdjustment
------------------------- ------------------------ -----------------
-0                        0                        -1
-
-PS C:\> Set-AASScalingPolicy -ServiceNamespace AppStream -PolicyName ASFleetScaleInPolicy -PolicyType StepScaling -ResourceId fleet/MyFleet -ScalableDimension appstream:fleet:DesiredCapacity -StepScalingPolicyConfiguration_AdjustmentType ChangeInCapacity -StepScalingPolicyConfiguration_Cooldown 360 -StepScalingPolicyConfiguration_MetricAggregationType Average -StepScalingPolicyConfiguration_StepAdjustments $stepadjustment
+PS C:\> Set-AASScalingPolicy -ServiceNamespace AppStream -PolicyName ASFleetScaleInPolicy -PolicyType StepScaling -ResourceId fleet/MyFleet -ScalableDimension appstream:fleet:DesiredCapacity -StepScalingPolicyConfiguration_AdjustmentType ChangeInCapacity -StepScalingPolicyConfiguration_Cooldown 360 -StepScalingPolicyConfiguration_MetricAggregationType Average -StepScalingPolicyConfiguration_StepAdjustments @{ScalingAdjustment = -1; MetricIntervalUpperBound = 0}
```
Comment (matteo-prosperi) [inline generator/AWSPSGeneratorLib/HelpMaterials/Examples/AAS/Set-AASScalingPolicy.xml]: Please remove the line break after `<code>`.
This should look like
```
    <code>PS C:\>...
```

**Comment by ajsing** [comment]
Fixed formatting issues, please review.

**Comment by ashishdhingra** [comment]
PR merged

**Pull request closed without merging**